File: Clase 06-02-2016/Laboratorio Eder Gonzalez/Laboratorio Eder Gonzalez/Centralita.vb
Public Class Centralita
    Private Shared _Acumulador As Double
    Private Shared _Contador As Integer

    Public Shared Function GetTotalLlamadas() As Integer
        Return _Contador
    End Function

    Public Shared Function GetTotalFacturacion() As Double
        Return _Acumulador
    End Function

    Public Shared Sub RegistrarLlamada(ByVal Registro As Llamada)
        _Contador = _Contador + 1
        _Acumulador = _Acumulador + Registro.CalcularPrecio()
    End Sub



End Class

File: CentralitaC/CentralitaC/Centralita.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CentralitaC
{
    class Centralita
    {
        private double _Acumulador;
        private int _Contador;

        public int GetTotalLlamadas()
        {
            return _Contador;
        }

        public double GetTotalFacturacion()
        {
            return _Acumulador;
        }

        public void RegistrarLlamada(Llamada Registro)
        {
            _Contador = _Contador + 1;
            _Acumulador = _Acumulador + Registro.CalcularPrecio();
        }
    }
}
